collections/Lat_bib/MS_Lat_bib_d_11.xml

```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/bodleian/consolidated-tei-schema/master/msdesc.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://raw.githubusercontent.com/bodleian/consolidated-tei-schema/master/msdesc.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xml:id="manuscript_10026" xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader>
+      <fileDesc>
+         <titleStmt>
+            <title>MS. Lat. bib. d. 11</title>
+            <title type="collection">MSS. Lat. bib. (Latin bibles)</title>
+            <respStmt>
+               <resp>Cataloguer</resp>
+               <persName>Laura Light</persName>
+               <persName>Andrew Dunning</persName>
+            </respStmt>
+         </titleStmt>
+         <publicationStmt>
+            <publisher>
+               <orgName type="department">Special Collections</orgName>
+               <orgName type="unit">Bodleian Libraries</orgName>
+               <orgName type="institution">University of Oxford</orgName>
+               <email>specialcollections.enquiries@bodleian.ox.ac.uk</email>
+            </publisher>
+            <idno type="msID">MS_Lat_bib_d_11</idno>
+            <idno type="collection">Lat bib</idno>
+            <idno type="catalogue">Western</idno>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="MS_Lat_bib_d_10" xml:lang="en">
+               <msIdentifier>
+                  <country>United Kingdom</country>
+                  <region type="county">Oxfordshire</region>
+                  <settlement>Oxford</settlement>
+                  <institution>University of Oxford</institution>
+                  <repository>Bodleian Library</repository>
+                  <idno type="shelfmark">MS. Lat. bib. d. 11</idno>
+                  <altIdentifier type="internal">
+                     <idno type="SCN">Not in SC (late accession)</idno>
+                  </altIdentifier>
+               </msIdentifier>
+               <msContents>
+                  <textLang mainLang="la">Latin</textLang>
+                  <msItem>
+                     <locus>(fols. 1r–90v)</locus>
+                     <title key="work_12445" type="desc">Bible, Old Testament, Job, with gloss</title>
+                     <bibl type="bible">
```

Comment:
lines 45-7 don't do any harm (and are in other files I know) but aren't necessary now since subject entries are in works.xml